### PR TITLE
User/shubham90/uuid

### PR DESCRIPTION
**Pull request by shubham90**
I have added UUId to the Quiz. Also did some other refactors.

### CODE REVIEW

File: service/bookturks/quiz/QuizTools.py [hunk 1]
```diff
@@ -22,24 +22,6 @@ def __init__(self):
...
-        """
-        Concatenate to form the quiz_id
-        :param username:
-        :param quiz_name:
-        :return:
-        """
-        if not quiz_name.rstrip():
-            raise ValueError("Quiz Name cannot be blank")
-        quiz_id = "_".join([username, quiz_name.lower()])
-        if re.match("^[A-Za-z0-9_ -]*$", quiz_name):
```
Comment (zalokhan) [inline service/bookturks/quiz/QuizTools.py]: Are we having these checks anywhere ?
If not we can add this as a priority TODO and add it later.
We can perform this check on the front-end too which will make it faster for the user.

Comment (shubham90) [inline service/bookturks/quiz/QuizTools.py]: Will add this server check as a new commit in my PR.

Comment (zalokhan) [inline service/bookturks/quiz/QuizTools.py]: Sounds good. Ill wait for the PR to merge then.

Comment (shubham90) [inline service/bookturks/quiz/QuizTools.py]: @zalokhan added the check in new commit https://github.com/zalokhan/BookTurks/pull/31/commits/82392109a9485fd65bcf4b26a0ac3ce231554b9a

Comment (shubham90) [inline service/bookturks/quiz/QuizTools.py]: Travis CI is not happy, I am not sure why

Comment (zalokhan) [inline service/bookturks/quiz/QuizTools.py]: Yes it crashes quite a lot in PRs. Should get solved after merging.
Ill fix any broken tests.

**Pull request merged**